src/components/Posts/PostItem.tsx

```diff
@@ -24,22 +24,33 @@ import { Post, updatePost } from "../../state/slices/postsSlice";
 import { useNavigate } from "react-router-dom";
 import { setExpandedPost } from "../../state/slices/expandedPostSlice";
 
-const styles = {
-  card: {
-    boxShadow: "none",
-  },
-  cardActions: {
-    width: "100%",
-  },
-};
-
 type PostProps = {
   post: Post;
 };
 
 const PostItem = ({ post }: PostProps) => {
   const dispatch = useAppDispatch();
   const user = useAppSelector((state) => state.user);
+  const styles = {
```

Comment:
I kinda don't like moving the styles inside the component like this.. I know you did it for the ternary statement, but if possible I wanna find a way to add this conditional styling while keeping the current pattern of separating our `styles` object from the component

Reply:
I think I figured out a neat way to get around this! Check out my next commit

Reply:
We found a workaround for now, if anyone finds something better we'll revisit this later